### PR TITLE
Mobile schedule

### PR DESCRIPTION
**Pull request by TheTincho**
This PR adds the necessary files to provide the schedule to giggity or similar applications.

It also adds a (very dirty and brittle) script that takes the markdown schedule and creates the xml schedule.

Once this is merged, I will ask  giggity author to add PromCon to the automatic list of events.

### CODE REVIEW

File: content/2018-munich/talks/panel-discussion-prometheus-long-term-storage-approaches.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ title: "Panel Discussion: Prometheus Long-Term Storage Approaches"
 
 ## Panel Discussion: Prometheus Long-Term Storage Approaches
 
-In this panel, we will explore some of the emerging and competing approaches to Prometheus scalable long-term storage.
```
Comment (RichiH) [inline content/2018-munich/talks/panel-discussion-prometheus-long-term-storage-approaches.md]: This was moved here on purpose as it's the only one where you need context before reading through the list of people & projects.

Comment (TheTincho) [reply content/2018-munich/talks/panel-discussion-prometheus-long-term-storage-approaches.md]: Ah, I thought it was a mistake (and also broke the very crude regex-based parsing :)). I will revert it.

File: content/2018-munich/schedule.xml [hunk 2]
```diff
@@ -467,13 +480,14 @@ we will have a write-in poster for people interested in giving a LT.
...
-        <duration>0:30</duration>
+        <duration>00:30</duration>
         <room>Main room</room>
         <abstract/>
         <title>Improving Reliability Through Engineering an Easy-to-use Prometheus-Based Monitoring and Alerting Stack: Introducing Our Reliability Toolkit</title>
         <type>talk</type>
         <released>True</released>
         <persons>
+          <person>Robin van Zijll</person>
           <person>Janna Brummel (ING N.V.)</person>
```
Comment (RichiH) [inline content/2018-munich/schedule.xml]: Janna has been removed as a speaker; where did your script pull that info?

**Comment by RichiH** [comment]
I removed Janna with https://github.com/promcon/website/pull/83

File: content/2018-munich/schedule.xml (hunk 2 above)
Comment (TheTincho) [reply content/2018-munich/schedule.xml]: From `content/2018-munich/schedule.md`, Janna is still listed there

**Comment by TheTincho** [comment]
I removed her from the MD schedule, and re-run the script.

**Comment by TheTincho** [comment]
PTAL?

**Comment by RichiH** [comment]
Thanks!

**Pull request merged**